public/js/index.js

```diff
@@ -0,0 +1,24 @@
+//Youtube slideshow for index page
+function onGoogleLoad() {
+  gapi.client.setApiKey('AIzaSyCzI5Gvpq1hPlfjvB81EVc0vLv_11eW7NI');
```

Comment:
Yeah, it's a public one tho, however I think it's on my Youtube account. I guess I'll ask Tom for a URY one.